symfony/plugins/orangehrmCoreOAuthPlugin/entity/OAuthClient.php

```diff
@@ -0,0 +1,146 @@
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="ohrm_oauth_client")
+ * @ORM\Entity
+ */
+class OAuthClient
+{
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="client_id", type="string", length=80, nullable=false)
+     * @ORM\Id
+     */
+    private string $clientId;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="client_secret", type="string", length=80, nullable=false)
+     */
+    private string $clientSecret;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="redirect_uri", type="string", length=2000, nullable=false)
+     */
+    private string $redirectUri;
+
+    /**
+     * @return string
+     */
+    public function getClientId(): string
+    {
+        return $this->clientId;
+    }
+
+    /**
+     * @param string $clientId
+     */
+    public function setClientId(string $clientId): void
+    {
+        $this->clientId = $clientId;
+    }
+
+    /**
+     * @return string
+     */
+    public function getClientSecret(): string
+    {
+        return $this->clientSecret;
+    }
+
+    /**
+     * @param string $clientSecret
+     */
+    public function setClientSecret(string $clientSecret): void
+    {
+        $this->clientSecret = $clientSecret;
+    }
+
+    /**
+     * @return string
+     */
+    public function getRedirectUri(): string
+    {
+        return $this->redirectUri;
+    }
+
+    /**
+     * @param string $redirectUri
+     */
+    public function setRedirectUri(string $redirectUri): void
+    {
+        $this->redirectUri = $redirectUri;
+    }
+
+    /**
+     * @return string
+     */
+    public function getScope(): string
+    {
+        return $this->scope;
+    }
+
+    /**
+     * @param string $scope
+     */
+    public function setScope(string $scope): void
+    {
+        $this->scope = $scope;
+    }
+
+    /**
+     * @return string
+     */
+    public function getGrantTypes(): string
+    {
+        return $this->grantTypes;
+    }
+
+    /**
+     * @param string $grantTypes
+     */
+    public function setGrantTypes(string $grantTypes): void
+    {
+        $this->grantTypes = $grantTypes;
+    }
+
+    /**
```

Comment:
move below properties to top of getter/setter method declaration

Reply:
updated